lib/team_statistics.rb

```diff
@@ -0,0 +1,200 @@
+require_relative "game_data"
+require_relative "team_data"
+require_relative "game_team_data"
+require_relative "league_statistics"
+require "csv"
+
+class TeamStatistics < LeagueStatistics
```

Comment:
Using inheritance, love it.

Reply:
Can't wait to fully practice this as we refactor!